index.html

```diff
@@ -24,8 +26,13 @@
   <!-- Section Introduction and Social Media OBS> Improve naming First section etc..-->
 
   <section class="intro-section">
-    <h1 class="intro-section__title"><div>Hey there. I'm Glauber<br></div>
-    <div class="intro-section__subtitle">I'm a software developer<br></h1>
+    <nav class="desktop-nav-bar">
+      <li>Portfolio</li>
+      <li>About</li>
+       <li>Contact</li>
+     </nav>
+    <header class="intro-section__title">Hey there. I'm Glauber</header>
+    <div class="intro-section__subtitle">I'm a software developer</div>
```

Comment:
- [OPTIONAL] Kindly take a read on [this article](https://www.reliablesoft.net/h1-tag/) for a better understanding of the `h1` tag. The main title of the page is wrapped with the `h1` tag and the page should have one but no more than one `h1` tag. The `header` tag is not the better tag to wrap the main title, as you used above to wrap the navbar of the mobile version. Also, the navbar of the desktop version would probably fit better in that `header` tag above the first section. Also what is wrapped with the `div` tag seems like part of the main title of the page. Kindly be aware of that.